PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+#### What does this PR do?
+
+Fixes #(issue_number)
+
+#### Which TC components are affected by this PR?
+
+- [ ] Grove
+- [ ] Traffic Analytics
+- [ ] Traffic Monitor
+- [ ] Traffic Ops
+- [ ] Traffic Ops ORT
+- [ ] Traffic Portal
+- [ ] Traffic Router
+- [ ] Traffic Stats
+- [ ] Traffic Vault
+- [ ] Other _________
+
+#### What is the best way to verify this PR?
+
+
+#### Check any that apply
+
+- [ ] This PR does *NOT* fix a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)
+- [ ] This PR includes tests
+- [ ] This PR includes documentation updates
+- [ ] This PR includes an update to CHANGELOG.md
+
```

Comment:
does it need to mention newly introduced files that require a license header (like this one?) :-)

Reply:
@dangogh - i was thinking that as well. how about another checkbox like:

- [ ] This PR includes all required license headers

Reply:
I added the license header and a bullet for that